generatebundlefile/hack/pullpush.sh

```diff
@@ -19,6 +34,9 @@ aws ecr-public describe-images --region us-east-1 --repository-name cert-manager
 aws ecr-public create-repository --repository-name cert-manager-controller
 aws ecr-public create-repository --repository-name eks-anywhere-test
 
+aws ecr-public create-repository --repository-name eks-anywhere-packages
```

Comment:
Two things here. First is that with ecr-public, it's best to manually set the region. This is because ecr-public only supports us-east-1 for whatever reason. 

```sh
aws ecr-public create-repository --region us-east-1 --repository-name eks-anywhere-test
```

The second thing is that this will fail if the repo already exists. I have a bunch of scripts I use to push/pull packages and I do something like this:

```sh

function repoExists () {
    aws "${AWS_CMD[*]}" describe-repositories \
        | jq -r ".repositories[].repositoryName" \
        | grep -q "^$1\$"
}
```

Then later in the script...

```sh
        if ! repoExists "${REPO}"; then
            aws "${AWS_CMD[*]}" create-repository --repository-name "${REPO_BASE#oci://}"
        fi
```

Reply:
The hack isn't for part of the pipelines, but I can make some changes to make it more user friendly.

Reply:
If it isn't for the pipelines, then all the more reason to not assume anything about the environment.